vmware-tanzu/tanzu-cli
# Fix the file locking issue on Windows

### What this PR does / why we need it

- For more details see: https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/204

### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR
- Build the Tanzu CLI for Windows, Mac, Linux with the fix
- Verify that CLI can install and run the `package` and `space` plugin (built with older Tanzu Plugin Runtime)

- Build the `space` plugin using v1.4.2 version of Tanzu Plugin Runtime
- Verify that CLI can install the `space` plugin (built with newer v1.4.2 Tanzu Plugin Runtime)

**Windows Tests:**
![Screenshot 2024-08-08 at 2 54 54 PM](https://github.com/user-attachments/assets/a440ab98-09a7-424c-8efb-4d2ae8398e42)





### Release note

```release-note
Fix the file locking issue on Windows
```



### Additional information

#### Special notes for your reviewer
